### DISCUSSION TITLE
switch interfaces

### DISCUSSION THREAD

**Discussion by PiranhaBgl** [travisghansen/hass-pfsense#114] [Ideas]
hello,
possible to add a switch for shut or up an interfaces?

**Comment by travisghansen** [comment]
No but it would be pretty easy to use a service to do that. You just want to down the interface?

**Reply by virtualuk** [reply]
I'm looking to do the same thing, any chance you could show how it's done with a service, please?  Looking to effectively be able to have the ability to bring up and down and interface in pfSense from HA. Thanks!
